### DISCUSSION TITLE
sdk problem for installation the package

### DISCUSSION THREAD

**Discussion by SirKafi15** [dotMorten/WinUIEx#106] [Q&A]
Hey. i wondering how to get the solution for this one. I wanted to install the latest version package of WinUIEx from NuGet for my WinUI 3 project. but my sdk is the latest version . so the error from the console like this :

NuGet\Install-Package : NU1605: Detected package downgrade: Microsoft.WindowsAppSDK from 1.2.221109.1 to 1.1.3. Reference the package directly from the project to select a different version.

It would be nice to get the answer from you. thankyou..

**Comment by dotMorten** [comment]
Your application is referencing v1.1.3 of the windows app Sdk, but the latest WinUIEx requires 1 .2.221109.1+

**Reply by SirKafi15** [reply]
but mine already v1.2 (latest). i don't know what's wrong with it

**Reply by SirKafi15** [reply]
I still can't install it. how to make it happen sir? thankyou

**Reply by dotMorten** [reply]
Hard to tell what your issue is, but it isn’t a WinUIEx issue but an issue with your nuget packages or your development environment being in a bad state. Clear out you obj folders and force a restore, check all your project’s package references etc
